numpy/core/_add_newdocs.py

```diff
@@ -2235,6 +2247,15 @@
 add_newdoc('numpy.core.multiarray', 'ndarray', ('__array_struct__',
     """Array protocol: C-struct side."""))
 
+add_newdoc('numpy.core.multiarray', 'ndarray', ('__dlpack__',
+    """a.__dlpack__(self, /, *, stream=None)
```

Comment:
```suggestion
    """a.__dlpack__(*, stream=None)
```
Looking the other methods, it seems to be convention to omit the `self` parameter when describing method signatures.
https://github.com/numpy/numpy/blob/0891c5f659a4ddbe952e124410a7d40a3cbd234b/numpy/core/_add_newdocs.py#L2793